README.md

```diff
@@ -122,6 +123,32 @@ Registrations are not currently supported and will be addressed in J#PROJ-988.
 - cmd: node_modules/.bin/appveyor-commit-message-checker
 ```
 
+### As a commit hook
+
+A `commit-msg` hook is provided which you can setup to run in your development environment, to help catch invalid commit
+messages as they happen.
+
+This is intended to be setup in addition to, and definitely not instead of, checks that run [as part of CI](#as-part-of-ci).
+It's primary purpose is to shorten the feedback time between commiting something, and finding out that the commit
+message is invalid. The alternative being that you don't find out until it fails on CI, which could take a frustrating
+amount of time.
+
+To use the commit hook, copy or symlink the file to `.git/hooks/commit-msg` within your project. i.e. run:
```

Comment:
I actually have no idea how you would do that. I didn't realise you *could* do that. So no it's definitely not assumed, just overlooked.